content/en/blog/2023/any-metric-receiver.md

```diff
@@ -0,0 +1,480 @@
+---
+title: Receive any custom metric with the OpenTelemetry Collector
+linkTitle: Any Metric Receiver
+date: 2023-11-28
+author: '[Severinq Neumann](https://github.com/svrnm), Cisco'
+# prettier-ignore
+cspell:ignore: openssl servername noout enddate gomod debugexporter otlpexporter otlphttpexporter transformprocessor otlpreceiver carbonreceiver ottl datapoint webserver Helmuth
```

Comment:
We should add OTTL to the textlinter rules. WDYT?

Reply:
yes! Let's maybe do a new round of reviewing the ignore lists and see which ones we want to add